encoder/vaapiencoder_base.cpp

```diff
@@ -484,13 +539,14 @@ bool VaapiEncoderBase::initVA()
         return false;
     }
 
+    attrib[attribCount].type = VAConfigAttribEncROI;
```

Comment:
Would better set it here, if driver can not support VAConfigAttribEncROI, we can get error status from vaCreateConfig.